src/frontends/pytorch/src/op/tensordot.cpp

```diff
@@ -0,0 +1,32 @@
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/matmul.hpp"
+#include "openvino/op/reduce_sum.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_tensordot(const NodeContext& context) {
+    num_inputs_check(context, 3, 4);
+
+    auto a = context.get_input(0);
+    auto b = context.get_input(1);
+    auto matmul_result = context.mark_node(std::make_shared<v0::MatMul>(a, b, false, false));
+
+    if (!context.input_is_none(2)) {
+        auto dims_a = context.get_input(2);
+        auto reduce_sum_result = context.mark_node(std::make_shared<v1::ReduceSum>(matmul_result, dims_a, true));
```

Comment:
This looks not meeting the definition about the input 2 defined here:
https://docs.pytorch.org/docs/stable/generated/torch.tensordot.html

It doesn't interpret neither 'int' dims nor 'Tuple' dims correctly to me.
Can you elabrate?